api/proto/canary/google.proto

```diff
@@ -32,32 +32,36 @@ message GoogleAccount {
   // The name of the account.
   string name = 1;
 
-  // The path to a JSON service account that Spinnaker will use as credentials.
-  // This is only needed if Spinnaker is not deployed on a Google Compute Engine
-  // VM, or needs permissions not afforded to the VM it is running on. See
-  // https://cloud.google.com/compute/docs/access/service-accounts for more information.
+  // The path to a JSON file containing the service account key that Spinnaker
+  // will use to authenticate.
+  // 
+  // You need this only if Spinnaker is not deployed on a Google Compute Engine
+  // VM, or if the account needs permissions not afforded to the VM it is
+  // running on.
+  // See https://cloud.google.com/compute/docs/access/service-accounts for more information.
   string jsonPath = 2;
 
-  // The name of a storage bucket that this account has access to. If you
-  // specify a globally unique bucket name that doesn't exist yet, Kayenta will
-  // create that bucket for you.
+  // The name of a Cloud Storage bucket that this account has access to. If you
+  // specify a globally unique bucket name that doesn't exist yet, Kayenta 
+  // creates that bucket for you.
   string bucket = 3;
 
-  // This is only required if the bucket you specify doesn't exist yet. In that
-  // case, the bucket will be created in that location. See
+  // Where to create the new bucket. This is only required if the bucket you
+  // specify doesn't exist yet. See
   // https://cloud.google.com/storage/docs/managing-buckets#manage-class-location.
   string bucketLocation = 4;
 
-  // The root folder in the chosen bucket in which to store all of the canary
-  // service's persistent data in. Defaults to `kayenta`.
+  // The root-level folder, in the specified bucket, in which to store all
+  // the canary service's persistent data. Defaults to `kayenta`.
   google.protobuf.StringValue rootFolder = 5;
 
   // (Required) The Google Cloud Platform project the canary service will use to
-  // consume GCS and Stackdriver.
+  // consume telemetry.
```

Comment:
Does this make it less clear that the project could also be used for GCS in addition to Stackdriver-supplied telemetry data?